### PR TITLE
Feature/some output fixes and cosmetics

### PR DESCRIPTION
**Pull request by mxmCherry**
Partially addresses https://github.com/bsm/grape-apidoc/issues/8

Everything done, except Table of Contents. And, well, changing `Mock::Bar` -> `Mock:Bar` (I don't like that change, let's keep it more ruby-ish, it's nice and clear already).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,11 +53,27 @@ class SomeAPI < Grape::API do
...
+  post('/bars/:id') do
+    Mock::Bar.new(
+      bar_id: 2,
+      foos: [Mock::Foo.new(foo_id: 1)],
+    )
+  end
 end
 ```
+
+Example of produced apidoc for this app: [spec/fixtures/golden.md](spec/fixtures/golden.md)
```
Comment (mxmCherry) [inline README.md:79]: Can we have the mock app (entities, api) somewhere there as well? (linkable, don't think it's OK to place ruby classes in fixtures dir)

File: spec/fixtures/golden.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
 # Entities
 
-## Mock::Bar::Entity
+## Mock::Bar
```
Comment (dim) [inline spec/fixtures/golden.md:3]: can we replace all `::` with just `:` - it makes more sense for the doc + makes better anchors

**Review by dim — APPROVED** [review]

File: spec/fixtures/golden.md (hunk 2 above)
Comment (mxmCherry) [reply spec/fixtures/golden.md:3]: I don't like it at all, but OK.

File: README.md (hunk 1 above)
Comment (mxmCherry) [reply README.md:79]: @dim any suggestions? ^^

Comment (dim) [reply README.md:79]: you can do what I did in grape-app and have a `spec/scenario`?

Comment (mxmCherry) [reply README.md:79]: I'll try, but in another PR.

Comment (mxmCherry) [reply README.md:79]: https://github.com/bsm/grape-apidoc/pull/11

**Pull request merged**